qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/AppointmentBlocksPage.java

```diff
@@ -19,6 +19,7 @@
 public class AppointmentBlocksPage extends Page {
 
 	public static final By DELETE = By.linkText("Delete");
+	public static final By CURRENT_DAY_BUTTON = By.cssSelector("#calendar table tbody tr td.fc-header-left span.fc-button.fc-button-basicDay.fc-state-default.fc-corner-left.fc-corner-right");
```

Comment:
Can we use a different element , probably xpath. this is likely not convinient.

Reply:
@mherman22 I don't know if this answers your question. I think line 8 does that.

Reply:
> @mherman22 I don't know if this answers your question. I think line 8 does that.

yeah after reading through i saw it

Reply:
> Can we use a different element , probably xpath. this is likely not convinient.

@sherrif10  earlier on like a few months back, I used xpath and this was causing the test to fail until I switched to css selector. However, let me give it another try. I will give you feedback.

Reply:
@sherrif10  I have attended to the review and pushed a commit. You could take some time to make more reviews.

Reply:
I am sorry being late in the conversation here.

> Can we use a different element , probably xpath. this is likely not convinient

@sherrif10 I find the css locator being more reliable than using xpath so I would go with the locator suggested by @Irenyak1

Reply:
No objection still, as long as it won't break in future